pokedex-with-bit/tests/test_pokedex.py

```diff
@@ -20,6 +20,10 @@ def config(tmpdir):
     os.environ['JINA_WORKSPACE'] = str(tmpdir)
 
 
+def sh():
```

Comment:
u can add the config and download as part of fixtures from which the test actually depends to better separate the preparation from the actual test

Reply:
maybe a better name? download_model?